src/csvcubed/utils/sparql_handler/data_cube_inspector.py

```diff
@@ -188,3 +225,71 @@ def get_code_lists_and_cols(self, csv_url: str) -> CodelistsResult:
         """
 
         return self._codelists_and_cols.get(csv_url, CodelistsResult([], 0))
+
+    def get_column_component_info(self, csv_url: str) -> List[ColumnComponentInfo]:
+        """
+        Get the ColumnComponentInfo which contains the EndUserColumnType
```

Comment:
This isn't quite right. What sort of comment do you think would be helpful here if you were a developer trying to figure out what this API does?